rune/dist/deb/debian/control

```diff
@@ -2,7 +2,7 @@ Source: rune
 Section: devel
 Priority: extra
 Maintainer: shirong <shirong@linux.alibaba.com>
-Build-Depends: debhelper (>=9), libseccomp-dev
+Build-Depends: debhelper (>=9), libseccomp-dev, libsgx-dcap-quote-verify, libsgx-dcap-quote-verify-dev
```

Comment:
If you change this, you need to make sure that yamls in workflows that generate rune deb and rpm works. For example: commit_crictl.yml, nightly-centos-sgx1.yml and so no

Reply:
> If you change this, you need to make sure that yamls in workflows that generate rune deb and rpm works in host. (nightly-centos-sgx1.yml, nightly-ubuntu-sgx1.yml).

Done, thanks.